### DISCUSSION TITLE
robin_nano35.bin not updating?

### DISCUSSION THREAD

**Discussion by dannler** [Guilouz/Klipper-Flsun-Speeder-Pad#55] [Q&A] [answered]
im on the step where i have to copy robin_nano35.bin to sd card and put it into printer

when sd card is inserted into printer anr started up, i do get the green update bar, but it takes way longer than the mentioned seconds to complete, and after the bar is fully green and i confirm if it was successfully installed i dont get the name change to robin_nano35.bin.cur

what is happening here?

**Comment by dannler** [comment, marked as answer]
also the time adjustment and location adjustment resets to the stock setting after i remove the sd card and switch the speederpad back on?

**Reply by cyb1337** [reply]
The SD-Card is supposed to go into the motherboard of the printer wich is not the same thing as the Speeder-Pad. The motherboard is located in the upper housing of the printer. In order to get to it you need to remove the 6x M3 screws at the top. There will be an SD-Card reader located on the Motherboard.

**Reply by dannler** [reply]
thank you

**Reply by stephenpfaff** [reply]
Came here to ask this, thanks @cyb1337
